src/Domain/Tags.hpp

```diff
@@ -0,0 +1,40 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+/// \file
+/// Defines tags related to domain quantities
+
+#pragma once
+
+#include <memory>
+
+#include "DataStructures/DataBoxTag.hpp"
+#include "DataStructures/Index.hpp"
+#include "Domain/CoordinateMaps/CoordinateMap.hpp"
+#include "Domain/Element.hpp"
+
+namespace Tags {
+
+/// The ::Element associated with the DataBox
+template <size_t VolumeDim>
+struct Element : db::DataBoxTag {
```

Comment:
Since we now have a `dg` namespace I'm wondering if this tag should be in `dg::Tags` and that we later move `Element` to `dg::Element`. Thoughts on these two changes?

Reply:
An `Element` is generic, and not restricted to DG